mne/bem.py

```diff
@@ -2187,3 +2187,62 @@ def check_seghead(surf_path=op.join(subj_path, 'surf')):
             incomplete=incomplete, extra=msg)
         write_bem_surfaces(dec_fname, dec_surf, overwrite=overwrite)
     logger.info('[done]')
+
+
+@verbose
+def distance_to_bem(pos, bem, trans=None, verbose=None):
+    """Calculate the distance of positions to inner skull surface.
+
+    Parameters
+    ----------
+    pos : array, shape (3,) | (n, 3)
+        Position(s) in m, in head coordinates.
+    bem : instance of ConductorModel
+        Conductor model.
+    %(trans)s If None (default), assumes bem is in head coordinates.
+    %(verbose)s
+
+    Returns
+    -------
+    distances : float | array, shape (n,)
+        The computed distance(s). A float is returned if pos is
+        an array of shape (3,) corresponding to a single position.
+    """
+    ndim = pos.ndim
+    if ndim == 1:
+        pos = pos[np.newaxis, :]
+
+    n = pos.shape[0]
+    distance = np.zeros((n,))
+
+    logger.info(
+        f'Computing distance to inner skull surface for {n} positions...'
+    )
+
+    if bem['is_sphere']:
+        center = bem['r0']
+
+        if trans:
+            center = apply_trans(trans, center, move=True)
+        radius = bem['layers'][0]['rad']
+
+        distance = radius - np.linalg.norm(
+            pos - center, axis=1
+        )
+
+    else:  # is BEM
+        surface_points = bem['surfs'][0]['rr']
+
+        if trans:
+            surface_points = apply_trans(
+                trans, surface_points, move=True
+            )
+
+        distance = np.min(np.linalg.norm(
+            surface_points[:, np.newaxis] - pos, axis=-1), axis=0
+        )
```

Comment:
In principle you could use `scipy.spatial.distance.cdist` here and `np.min` over that, but even better would be using our internal `_compute_nearest` with `return_dists=True` as it should be faster:

https://github.com/mne-tools/mne-python/blob/1349ad88bdfe72d9214fe7496f74b1629985673d/mne/surface.py#L474